ArchiSteamFarm/Steam/Data/AccessTokenResponse.cs

```diff
@@ -0,0 +1,17 @@
+
+using System.Diagnostics.CodeAnalysis;
+using Newtonsoft.Json;
+
+namespace ArchiSteamFarm.Steam.Data {
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	internal sealed class AccessTokenResponse : EResultResponse {
+		[JsonProperty(PropertyName = "data", Required = Required.Always)]
+		internal AccessTokenData AccessTokenData { get; private set; } = new();
+	}
+
+	[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
+	internal sealed class AccessTokenData {
+		[JsonProperty(PropertyName = "webapi_token", Required = Required.Always)]
+		internal string Webapi_token { get; private set; } = string.Empty;
```

Comment:
Rename variable to `WebAPIToken` for consistency with the codestyle. And nit: use `""` instead of `string.Empty` (since it's used in other places).